index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" 
+    content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/index.css" >
+    <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <h1>My Logo</h1>  
+    
+    <button id="menu-btn"></button>
+
+    <nav id="header-nav">
+      <ul>
+        <li><a href="">Portfolio</a></li>
+        <li><a href="">About</a></li>
+        <li><a href="">Contact</a></li>
+        <li><a href="">Mail</a></li>
+      </ul>
+    </nav> 
+
+  </header>
+
+  <main id="main">
+    <section id="headline">
+
+      <h1>Hey there.<br> I am Khanh.</h1>
+      <h2>I am a Software Engineer</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hesitate to contact me.</p>
+
+      <nav id="headline-nav">
+        <ul>
+          <li><a href="https://github.com/khanh-devos"><span></span></a></li>
+          <li><a href=""></a></li>
+          <li><a href=""></a></li>
+          <li><a href=""></a></li>
+          <li><a href=""></a></li>
+        </ul>
+      </nav>  
+    </section>
+
+
+
+  </main>
```

Comment:
|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/235173689-275ceaf2-7352-44f9-818b-99df48127577.png) | ![image](https://user-images.githubusercontent.com/77069011/235173626-c049cbaa-608f-45ee-9a2c-d28f0b07e3f1.png) |


You did a great job working on this section 🚀 However, please make sure to use the right font, I see that you used it in your CSS, however, they're not getting applied because they're not downloaded in your code. Please consider fixing this! 🤓 🔧